mlflow/pyfunc/__init__.py

```diff
@@ -318,106 +441,3 @@ def predict(*args):
             return result[result.columns[0]]
 
     return pandas_udf(predict, result_type)
-
-
-def save_model(dst_path, loader_module, data_path=None, code_path=(), conda_env=None,
```

Comment:
Please bring `save_model` and `load_model` back down here so it will be easy to compare difference.

Reply:
I've moved `save_model` and `log_model` to `mlflow.pyfunc.model`. The logical flow differs substantially from the old method. It seems easier to review it as a completely new implementation rather than a modification of existing conventions. Please let me know if you disagree.

Reply:
Nope this is cool!
The design doc and inline documentation makes it super easy to follow. Thank you.